automatic/intunewinapputil/update.ps1

```diff
@@ -0,0 +1,34 @@
+Import-Module AU
+
+$releases = "https://github.com/microsoft/Microsoft-Win32-Content-Prep-Tool/releases/latest"
+
+function global:au_GetLatest {
+  $download_page = Invoke-WebRequest -UseBasicParsing -Uri $releases
+
+  $urlRegex = '\.zip'
+  $url = $download_page.Links | Where-Object href -match $urlRegex | Select-Object -First 1 -ExpandProperty href
+
+  $version = ($url -split '/' | Select-Object -Last 1).Replace('v','').Replace('.zip','')
+
+  @{
+    Url32 = "https://github.com$url"
+    Version = $version
+  }
+
+}
+
+function global:au_BeforeUpdate {
+  Get-RemoteFiles -Purge -NoSuffix
+}
+
+function global:au_SearchReplace {
+  @{
+    "./tools/chocolateyInstall.ps1" = @{
+      "(?i)(^\s*url\s*=\s*)('.*)" = "`$1'$($Latest.URL32)'"
```

Comment:
Please intend this line so the equals sign is at the same position as the items below

Reply:
Updated this by just having VSCode format the file 😄